diff --git a/CMakeLists.txt b/CMakeLists.txt
index bee6cd581..448d7cb29 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -120,6 +120,8 @@ include(XercesWChar)
 include(XercesIconvConst)
 include(XercesLFS)
 
+set(BUILD_SHARED_LIBS OFF)
+
 # Generate headers
 configure_file(
   ${CMAKE_CURRENT_SOURCE_DIR}/config.h.cmake.in
@@ -149,15 +151,15 @@ set(includedir "${CMAKE_INSTALL_FULL_INCLUDEDIR}")
 configure_file(${CMAKE_CURRENT_SOURCE_DIR}/xerces-c.pc.in
                ${CMAKE_CURRENT_BINARY_DIR}/xerces-c.pc)
 install(
-  FILES ${CMAKE_CURRENT_BINARY_DIR}/xerces-c.pc
-  DESTINATION "${PKGCONFIGDIR}"
-  COMPONENT "development")
+ FILES ${CMAKE_CURRENT_BINARY_DIR}/xerces-c.pc
+ DESTINATION "${PKGCONFIGDIR}"
+ COMPONENT "development")
 
 # Process subdirectories
-add_subdirectory(doc)
+#add_subdirectory(doc)
 add_subdirectory(src)
-add_subdirectory(tests)
-add_subdirectory(samples)
+#add_subdirectory(tests)
+#add_subdirectory(samples)
 
 # Display configuration summary
 message(STATUS "")
diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index a168db125..d590735cf 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -1075,12 +1075,12 @@ endif()
 list(APPEND libxerces_c_SOURCES ${xerces_replacement_funcs})
 
 # NetAccessors, conditionally built based on selection
-if(SOCKET_LIBRARY)
-  list(APPEND libxerces_c_DEPS ${SOCKET_LIBRARY})
-endif()
-if(NSL_LIBRARY)
-  list(APPEND libxerces_c_DEPS ${NSL_LIBRARY})
-endif()
+# if(SOCKET_LIBRARY)
+#   list(APPEND libxerces_c_DEPS ${SOCKET_LIBRARY})
+# endif()
+# if(NSL_LIBRARY)
+#   list(APPEND libxerces_c_DEPS ${NSL_LIBRARY})
+# endif()
 
 if(XERCES_USE_NETACCESSOR_CURL)
   list(APPEND libxerces_c_SOURCES ${curl_sources})
